libraries/c_sdk/standard/mqtt/src/iot_mqtt_network.c

```diff
@@ -661,7 +676,7 @@ static IotMqttError_t _deserializeIncomingPacket( _mqttConnection_t * pMqttConne
             }
             else
             {
-                EMPTY_ELSE_MARKER;
+                status = IOT_MQTT_BAD_RESPONSE;
```

Comment:
please add comments about why this status is assigned here.

Reply:
Addressed